wiki/Welcome/en.md

```diff
@@ -0,0 +1,31 @@
+# Welcome
+
+Welcome to _osu!_, a free-to-win rhythm game—developed by peppy—with four game modes: osu!standard, a circle clicking simulator; osu!taiko, a drum drumming emulator; osu!catch, a fruit salad catcher; osu!mania, a key smashing mania.
+
+## Current events
```

Comment:
links to news page? or we have to update it once a month ;/

Reply:
emphasis stated this in ppy#94

Reply:
also, news isn't really helpful in knowing what actually is going on